src/app/core/upload/upload.service.ts

```diff
@@ -86,41 +92,42 @@ export class Upload {
     this.upload();
   }
 
-  cancel(): boolean {
-    if (this.evaporate && !this.complete) {
-      let formerId = this.uploadId;
-      this.uploadId = null;
-      if (formerId !== null) {
-        return this.evaporate.cancel(formerId);
+  cancel(): Observable<boolean> {
```

Comment:
I think cancelling in-progress uploads is no longer working.  In current staging, I see it actually cancel the PUT xhr requests, and issue a DELETE to cloudfront.  On this branch, the PUTs continue, and then it does a POST at the end to complete the upload.

Reply:
Fixed